spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/ConnectorConstants.java

```diff
@@ -5,8 +5,15 @@
 
 package com.datastrato.gravitino.spark.connector;
 
+import com.datastrato.gravitino.rel.expressions.sorts.SortDirection;
+
 public class ConnectorConstants {
   public static final String COMMENT = "comment";
 
+  public static final SortDirection SPARK_DEFAULT_SORT_DIRECTION = SortDirection.ASCENDING;
+  public static final String LOCATION = "location";
```

Comment:
I would suggest you using the `LOCATION` in the module `bundled-catalogs`

Reply:
Is I miss something? I couldn't find `LOCATION` or any properties defined in `bundled-catalogs`

Reply:
`LOCATION` is defined in the Hive catalog module and can't be readable through `bundled-catalogs`.

Reply:
the properties are now defined in XXcatalogs and used by spark and trino, I prefer a separate issue to track this.

Reply:
use #2617 to track this